src/helpers/graph.js

```diff
@@ -1,38 +1,45 @@
-export function removeNoteFromGraph(nodes, edges, noteToDelete) {
+function doesDeletedNodeHasSiblings(edges, parentId) {
+  return edges.find(edge => edge.from === parentId);
+}
+
+function revokeParentStatus(nodes, parentId) {

```

Comment:
For `doesDeletedNodeHasSiblings` one has to pass `edges` as the first parameter.

For `revokeParentStatus` one has to pass `nodes` as the first argument.

I think it might be confusing when using these functions.

Maybe would be more handy to make it like `doesDeletedNodeHasSiblings({ edges, parentId })` and `revokeParentStatus({ nodes, parentId })`?

Reply:
Also, why does it accept `parentId` ? Is it revoking parent status from the passed parent, or from a child of this parent?

I think it would be less confusing if it was `revokeParentStatus({ nodes, nodeId })`. So we understand that the parent status is revoked from the passed node.

Reply:
Done